src/ansys/fluent/core/fluent_connection.py

```diff
@@ -201,6 +201,84 @@ def list_values(self) -> dict:
         return vars(self)
 
 
+def _get_ip_and_port(
+    ip: Optional[str] = None, port: Optional[int] = None
+) -> (str, int):
+    if not ip:
+        ip = os.getenv("PYFLUENT_FLUENT_IP", "127.0.0.1")
+    if not port:
+        port = os.getenv("PYFLUENT_FLUENT_PORT")
+    if not port:
+        raise PortNotProvided()
+    return ip, port
+
+
+def _get_channel(ip: str, port: int):
+    # Same maximum message length is used in the server
+    max_message_length = _get_max_c_int_limit()
+    return grpc.insecure_channel(
+        f"{ip}:{port}",
+        options=[
+            ("grpc.max_send_message_length", max_message_length),
+            ("grpc.max_receive_message_length", max_message_length),
+        ],
+    )
+
+
+class _ConnectionInterface:
+    def __init__(self, create_grpc_service, error_state):
+        self._scheme_eval_service = create_grpc_service(SchemeEvalService, error_state)
+        self.scheme_eval = service_creator("scheme_eval").create(
+            self._scheme_eval_service
+        )
+
+    @property
+    def product_build_info(self) -> str:
+        """Get Fluent build info."""
+        build_time = self.scheme_eval.scheme_eval("(inquire-build-time)")
+        build_id = self.scheme_eval.scheme_eval("(inquire-build-id)")
+        rev = self.scheme_eval.scheme_eval("(inquire-src-vcs-id)")
+        branch = self.scheme_eval.scheme_eval("(inquire-src-vcs-branch)")
+        return f"Build Time: {build_time}  Build Id: {build_id}  Revision: {rev}  Branch: {branch}"
+
+    def get_cortex_connection_properties(self):
+        """Get connection properties of Fluent."""
+        from grpc._channel import _InactiveRpcError
+
+        try:
+            logger.info(self.product_build_info)
+            logger.debug("Obtaining Cortex connection properties...")
+            fluent_host_pid = self.scheme_eval.scheme_eval("(cx-client-id)")
+            cortex_host = self.scheme_eval.scheme_eval("(cx-cortex-host)")
+            cortex_pid = self.scheme_eval.scheme_eval("(cx-cortex-id)")
+            cortex_pwd = self.scheme_eval.scheme_eval("(cortex-pwd)")
+            logger.debug("Cortex connection properties successfully obtained.")
+        except _InactiveRpcError:
+            logger.warning(
+                "Fluent Cortex properties unobtainable, force exit and other"
+                "methods are not going to work properly, proceeding..."
```

Comment:
```suggestion
                "Fluent Cortex properties unobtainable. 'force exit()' and other"
                "methods are not going to work properly. Proceeding..."
```
These should be short sentences and not run-on sentences. What does the concluding "Proceeding" mean? Proceeding with what?

Reply:
Thank you @PipKat. We'll get rid of it.